tutorials/core-concepts/users-and-permissions/managing-accesss-control.md

```diff
@@ -1,5 +1,159 @@
 ---
-title: Manaing Access Control
+title: Managing Access Control
 layout: tutorial
 ---
 
+# Managing Access Control
+
+In most cases, you essentially have three levels of permissions:
+
+* The administrator (`admin`), who can do anything
+
+* The editor or writer (`edit`), who can create content, but doesn't have access to configuration or management of the site
+
+* The guest (`guest`), who can look at all the cool stuff that everyone else makes
+
+Apostrophe makes it easy to set this up with minimal configuration, but if that isn't enough, you have have the option to create custom permissions schemes with more control over exactly who has access to what.
+
+## Standard Permissions
+
+Apostrophe ships with two user groups enabled: `admin` and `guest`. These are configured in the `index.js` for `apostrophe-user`. For some deployments, this might be all you need, but if you need just a little bit more, you can easily add `edit`. To add it, create a new configuration in `app.js` that overrides the default.
+
+### Example: Creating Group Permissions
+
+Since you're overriding an existing configuration, create the configuration for all three groups:
+
+* `guest`, with no permissions
+
+* `editor`, with the `edit` permission
+
+* `admin` with the `admin` permission.
+
+
+1. Open `app.js`
+
+2. Find the `modules` section and add an `apostrophe-users` block like the one below:
+
+{% code-tabs %}
+{% code-tabs-item title="app.js" %}
+```javascript
+  modules: {
+  	...
+    'apostrophe-users': {
+      groups: [
+        {
+          title: 'guest',
+          permissions: [ ]
+        },
+        {
+          title: 'editor',
+          permissions: [ 'edit' ]
+        },
+        {
+          title: 'admin',
+          permissions: [ 'admin' ]
+        }
+      ]
+    }
+    ...
+  }
+```
+{% endcode-tabs-item %}
+{% endcode-tabs %}
+
+![](/.gitbook/assets/user-add-editor.png)
+
+Now you can create new users in the *editor* group, which will have the ability to create new content and manage the content they create, but will lack the great power -- and great responsibility -- of the admin role.
+
+{% hint style='info' %}
+An admin can lock any piece type as `adminOnly` which will remove the ability of users with the `edit` permission (or otherwise customized permissions) to create or edit pieces of that type.
```

Comment:
This is done automatically for users and groups.

Reply:
Added this information to the sidebar.